src/Test/Utilities/Portable/Assert/AssertEx.cs

```diff
@@ -582,5 +582,47 @@ public static void Throws<TException>(Action action, Action<TException> checker
                 checker?.Invoke((TException)e);
             }
         }
+
+        public static void Equal(bool[,] expected, Func<int, int, bool> getResult, int size)
```

Comment:
>public static void Equal(bool[,] expected, Func<int, int, bool> getResult, int size) [](start = 8, length = 84)

Consider extracting a common helper that can be used for this method and `NullableReferenceTypes.AssertEqual()`:
```C#
static void Equal<T>(
    T[,] expected,
    Func<int, int, T> getResult,
    int size,
    Func<T, string> getItemAsString,
    string itemFormat);
```